PlayerAgent.cs

```diff
@@ -78,38 +100,43 @@ public override void CollectObservations(VectorSensor sensor)
 
         // 1 obserwacja
         sensor.AddObservation(Vector3.Distance(transform.localPosition, target.transform.localPosition));
-
-
     }
 
     public override void OnActionReceived(ActionBuffers actions)
     {
         var vectorForce = new Vector3();
         vectorForce.x = actions.ContinuousActions[0];
         vectorForce.z = actions.ContinuousActions[1];
-        
-
+       
         playerRigidbody.AddForce(vectorForce * speed);
 
         var distanceFromTarget = Vector3.Distance(transform.localPosition, target.transform.localPosition);
-
+        var x = distanceFromTarget; // nie wiem jak tu dac zmienna do tego
+        var mi = 0;
+        var sigma = 0.2;
+                            // nie wiem gdzie te funkcje wywolac
+        score(distanceFromTarget); // nie wiem co tu wsadzic jako parametr
+        normal_distribution(x, mi, sigma);
+        Debug.Log("Score: ");
+        Debug.Log(score(distanceFromTarget));
 
         // Nagradzanie agenta
 
-        AddReward(-1f / MaxStep);
-
         if (distanceFromTarget <= distanceRequired)
         {
             SetReward(1.0f);
             StartCoroutine(SwapGroundMaterial(successMaterial, 0.5f));
+         
             EndEpisode();
```

Comment:
```suggestion
            EndEpisode();
            return;
```
Chyba powinno się skończyć?